.prettierrc

```diff
@@ -1,5 +1,6 @@
 {
   "singleQuote": true,
   "printWidth": 88,
-  "proseWrap": "always"
+  "proseWrap": "always",
+  "trailingComma": "es5"
```

Comment:
:heart:

Reply:
again, on the "clean up the root" tip, could move these few lines into `package.json`.